int32 SUCCESS          = 0
int32 OBSTACLE_SONAR   = 1
int32 OBSTACLE_HOME    = 2
int32 OBSTACLE_TAG     = 3
int32 PATH_FAIL        = 4
int32 USER_ABORT       = 5
int32 TIMEOUT          = 6
int32 INSIDE_HOME      = 7
int32 OBSTACLE_CORNER  = 8

int32 result 
